lib/call-cabal-project-to-nix.nix

```diff
@@ -241,7 +241,7 @@ let
       chmod +w -R $out/.source-repository-packages
       rm -rf $out/.source-repository-packages
     fi
-    find $out -type f ! -name '*.nix' -delete
+    find $out \( -type f -or -type l \) ! -name '*.nix' -delete
```

Comment:
It turns out this was not necessary as I wound up copying `.cabal` files (in ghc-packages.nix`) rather than symlinking them (so I could run `sed` on them), but I definitely seems like a bug not to treat symlinks like regular files here.